### PR TITLE
OCPBUGS-3062: updates verification command and moves module

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.12 and 4.13

Issue:
https://issues.redhat.com/browse/OCPBUGS-21726
https://issues.redhat.com/browse/OCPBUGS-30262

Link to docs preview:
https://72953--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn#nw-ovn-ipsec-enable_configuring-ipsec-ovn

QE review:
- [x] QE has approved this change.

Additional information:
PTAL @anuragthehatter 
https://github.com/openshift/openshift-docs/pull/72354
https://github.com/openshift/openshift-docs/pull/72954

### CODE REVIEW

File: modules/nw-ovn-ipsec-enable.adoc [hunk 1]
```diff
@@ -23,3 +23,42 @@ As a cluster administrator, you can enable IPsec encryption after cluster instal
...
+--
+where:
+
+`<XXXXX>`:: Specifies the random sequence of letters for a pod from the previous step.
+--
++
+.Example output
+[source,text]
+----
+Defaulted container "northd" out of: northd, nbdb, kube-rbac-proxy, sbdb, ovnkube-master, ovn-dbchecker
```
Comment (anuragthehatter) [inline modules/nw-ovn-ipsec-enable.adoc:62]: probbaly purge this line to aligh with what we have shown in 4.12 and 4.13 changes?

File: modules/nw-ovn-ipsec-enable.adoc [hunk 2]
```diff
@@ -23,3 +23,41 @@ As a cluster administrator, you can enable IPsec encryption after cluster instal
...
+.Example output
+[source,terminal]
+----
+NAME                   READY   STATUS    RESTARTS   AGE
+ovnkube-master-fvtnh   6/6     Running   0          122m
+ovnkube-master-hsgmm   6/6     Running   0          122m
+ovnkube-master-qcmdc   6/6     Running   0          122m
+----
+
+. Verify that IPsec is enabled on your cluster:
```
Comment (adellape) [inline modules/nw-ovn-ipsec-enable.adoc:45]: You've got ", enter the following command:" in the previous step. I'd go with that (or switch both to "by running the following command:") here as well.

File: modules/nw-ovn-ipsec-enable.adoc [hunk 3]
```diff
@@ -23,3 +23,41 @@ As a cluster administrator, you can enable IPsec encryption after cluster instal
...
++
+[source,terminal]
+----
+$ oc -n openshift-ovn-kubernetes rsh ovnkube-master-<XXXXX> \
+  ovn-nbctl --no-leader-only get nb_global . ipsec
+----
++
+--
+where:
+
```
Comment (adellape) [inline modules/nw-ovn-ipsec-enable.adoc:55]: I'm sorta surprised this still worked w/o the continuation, but I would try adding this in here anyway for completeness.
```suggestion
+
```

**Comment by adellape** [comment]
@JoeAldinger Were you wanting this `enterprise-4.13` branch PR cherry-picked after merge to the `enterprise-4.12` branch? Or could you make a 4.12 PR?

File: modules/nw-ovn-ipsec-enable.adoc (hunk 2 above)
Comment (JoeAldinger) [reply modules/nw-ovn-ipsec-enable.adoc:45]: I was just moving this content, but I agree for consistency.

**Comment by JoeAldinger** [comment]
> @JoeAldinger Were you wanting this `enterprise-4.13` branch PR cherry-picked after merge to the `enterprise-4.12` branch? Or could you make a 4.12 PR?

I was going to merge this to 4.13 and then CP to 4.12 or manually CP the commit into a fresh 4.12 branch. What is the best way to do this?

File: modules/nw-ovn-ipsec-enable.adoc (hunk 3 above)
Comment (adellape) [reply modules/nw-ovn-ipsec-enable.adoc:55]: Oh sorry, nvm, I see now that this was in an open block `--`, so yeah, not necessary!

**Comment by kalexand-rh** [comment]
This looks like a contained enough change that you can try using the bot to CP back to 4.12.

**Pull request merged**